### ISSUE TITLE
Updates in real time MG4 

### ISSUE THREAD

**Issue by manulena** [SAIC-iSmart-API/saic-home-assistant-addon#68]
I've set up the MG4 with Home Assistant and it's going very well. The only issue I have is that it doesn't start updating the information unless I force it using the gateway refresh mode to force. I have the default values for all gateways. Is there a way to have it start updating as soon as the door is opened, for example?

I am starting with the design but I stopped as I found the starting of the update is not in real-time. 

Any help?
<img width="440" alt="Captura de pantalla 2024-03-05 a las 16 30 37" src="https://github.com/SAIC-iSmart-API/saic-home-assistant-addon/assets/115431266/1f8bcfec-1b07-4439-bfb9-0e53b6d3721f">

**Comment by Tonno87** [comment]
In short: No! 

The only active message from car to gateway is the engine start message. Opening or other things doesn't work because you need to pull the status first.
The only known way without battery drain is to open the car with mqtt / home assistant, so it sends a force command to get the actual status.

**Comment by Tonno87** [comment]
![image](https://github.com/SAIC-iSmart-API/saic-home-assistant-addon/assets/59800400/8c05b796-1892-4357-b7e2-361d5352478e)

Please check your addon logfile for the "check for new messages" job. Every 1 minute the gateway checks the mg servers for an engine-start-message. After a successful check the gateway starts polling every 30s (default), so there is nothing to do for you to start it.

**Comment by nanomad** [comment]
@manulena if the current add-on version doesn't work please try the legacy version (uninstall the new one first) from this same repo. If that version automatically refreshes the car state once you start driving then we may have a regression.

**Comment by manulena** [comment]
Tonno87, I found it. I had the value in 6000... that´s why the checking was every 1h40m. Now I updated the figure to 60s and it´s working. Now, I have to investigate why I have CommandNumber=4 while you have none. I cannot find where to play with these values.

<img width="1013" alt="Captura de pantalla 2024-03-07 a las 8 51 17" src="https://github.com/SAIC-iSmart-API/saic-home-assistant-addon/assets/115431266/5d952ea1-936f-489b-b6e8-3fa4dff97250">




Anyway, It´s working and I don´t want to bother you anymore!!!
It´s solved.
Thanks @Tonno87 and @nanomad for your support!!!!
